doc/quickstart/mapbender_quickstart.rst

```diff
@@ -433,14 +180,8 @@ What Next?
 
 This is only the first step on the road to using Mapbender. There is a lot more functionality you can try.
 
-Mapbender Website
-
-  https://mapbender.org/
-
-You find tutorials at
-
-  https://doc.mapbender.org
+* Mapbender Website https://mapbender.org/
 
-Get involved in the project
+* You find tutorials at https://doc.mapbender.org
```

Comment:
Maybe also add: You find an extended tutorial at https://doc.mapbender.org/en/quickstart.html

Reply:
done